src/psyclone/tests/gocean1p0_opencl_test.py

```diff
@@ -0,0 +1,181 @@
+# -----------------------------------------------------------------------------
```

Comment:
pylint gives 10/10 but says ...
I:170, 0: Useless suppression of 'invalid-name' (useless-suppression)

Reply:
Fixed.